srcpkgs/itch/files/io.itch.itch.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Type=Application
+Name=itch
+TryExec=itch
+Exec=itch %U
+Icon=itch
+Terminal=false
+Categories=Game;
+MimeType=x-scheme-handler/itchio;x-scheme-handler/itch;
+X-GNOME-Autostart-enabled=true
```

Comment:
Why autostart? I don't see why it should be.

Reply:
I followed the example of the source code! 

https://github.com/itchio/itch-setup/blob/4a138757685a092640aa5fbb49c6b8133c5325cf/native/native_linux.go#L480